qiskit/transpiler/coupling.py

```diff
@@ -134,6 +134,11 @@ def is_connected(self):
         except nx.exception.NetworkXException:
             return False
 
+    def neighbors(self, physical_qubit):
```

Comment:
I'm not sure if this behaves as expected:
```python
from qiskit.transpiler import CouplingMap
coupling_list = [[0, 1], [0, 2]]
coupling = CouplingMap(coupling_list)
physical_qubits = coupling.physical_qubits

print([qubit for qubit in coupling.neighbors(physical_qubits[1])])
```
```
[]
```
Shouldn't this be `[0]`?

Reply:
I suggest a test like this in `test/python/transpiler/test_coupling.py`: 
```python 

        coupling_list = [[0, 1], [0, 2]]
        coupling = CouplingMap(coupling_list)

        physical_qubits = coupling.physical_qubits
        self.assertEqual(set(coupling.neighbors(physical_qubits[0])), set([1, 2]))
```

Reply:
This is because the graph is directional. If you make it bidirectional then it will be a neighbor. I added a note and and a test.

Reply:
thanks! Adjusting and adding the test in 4b3a65b1a